BE/src/main/java/org/codesquad/team34/issuetracker/common/Status.java

```diff
@@ -0,0 +1,28 @@
+package org.codesquad.team34.issuetracker.common;
+
+import java.util.stream.Stream;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+@Accessors(fluent = true)
+@Getter
+public enum Status {
+    OPEN("open"), CLOSED("closed");
+
+    private final String label;
+
+    Status(String label) {
+        this.label = label;
+    }
+
+    public static Status fromLabel(String label) {
+        return Stream.of(Status.values())
+            .filter(status -> status.labelIsEqualTo(label))
+            .findFirst()
+            .orElse(null);
+    }
+
+    private boolean labelIsEqualTo(String label) {
+        return this.label.equals(label);
+    }
```

Comment:
여기는 굳이 분리하지 않고 사용해도 좋을 것 같습니다.

```suggestion
    public static Status fromLabel(String label) {
        return Stream.of(Status.values())
            .filter(status -> status.label.equals(label))
            .findFirst()
            .orElse(null);
    }
```

Reply:
동의합니다. enum은 상수 값 객체처럼 쓰이므로 필드값을 꺼내어 쓰더라도 설계상 문제는 없다고 생각합니다. Status 객체 내부에 존재하는 코드이기도 하구요. 

예전에 정확히 반대되는 취지의 피드백을 받은 적이 있어서 Status 객체에게 비교를 위임시켰는데, 현업자들 사이에서도 의견이 갈리는 부분인지요?

Reply:
개인 스타일 차이 정도로 보시면 될 것 같아요. ㅎㅎ 위 코드 같은 경우 저는 아래 이유들 때문에 굳이 라는 생각이 들었네요.
1. 따로 메서드로 빼지 않아도 이미 equals가 갖고있는 의미가 명확함
2. 중복되어 여러 곳에서 사용하지 않음

Reply:
이해했습니다. 감사합니다!